apps/service-portal/src/components/Sidebar/NavItem/NavItem.css.ts

```diff
@@ -94,9 +90,22 @@ export const text = style({
   overflow: 'hidden',
 })
 
+export const iconWrapper = style({
+  position: 'absolute',
+  right: 0,
+  paddingRight: theme.spacing[2],
+  paddingTop: theme.spacing[1],
+  cursor: 'pointer',
+  color: theme.color.blue600,
+  ':hover': {
+    color: theme.color.blue400,
+  },
+})
+
 export const icon = style({
   pointerEvents: 'none',
   height: '26px',
+  animation: `250ms ease-in-out 0s 1 ${iconEaseIn}`,
```

Comment:
willChange opacity here

Reply:
fixed - I promise to remember "will-change" next time 🥲